### PR TITLE
Fenix telemetry annotations for miscellaneous events

### PR DESCRIPTION
**Pull request by jonalmeida**
@Amejia481 and I used Mike's script to generate the files for each event based on the [classification sheet](https://docs.google.com/spreadsheets/d/1MG93qvZICmQ65IepOAikdhOYEc8fxjWZTiUY4oD9axk/edit#gid=0)

### CODE REVIEW

**Comment by jonalmeida** [comment]
@mcomella , we added a few more Performance tags so some more events that you might be interested in taking a look at. See: 6ddfcb1

**Review by badboy — APPROVED** [review]
I of course haven't checked each one individually, but if this is generated of the list then I think we should be good here.

It might make sense to go in and add some actualy notes if you have anything for individual metrics, but that's easier to do one by one.

**Comment by Amejia481** [comment]
> I of course haven't checked each one individually, but if this is generated of the list then I think we should be good here.
> 
> It might make sense to go in and add some actualy notes if you have anything for individual metrics, but that's easier to do one by one.

Thanks for the review :) , before landing, we will wait for @mcomella's  review.

**Comment by wlach** [comment]
Taking myself off of the review due to PTO :)

**Comment by Amejia481** [comment]
Sorry, about that I just added you for visibility for when you come back, hope you keep enjoying your PTO 🌴

File: annotations/fenix/metrics/first_session.adgroup/README.md
Comment (mcomella) [inline annotations/fenix/metrics/first_session.adgroup/README.md:6]: These `first_session` metrics don't seem to be particularly performance-y.

**Review by mcomella — APPROVED** [review]
I don't feel too strongly about what goes in the `Performance` category and what doesn't as I think we should collectively own performance but I commented on the one that didn't quite fit in with the others.

File: annotations/fenix/metrics/first_session.adgroup/README.md
Comment (jonalmeida) [reply annotations/fenix/metrics/first_session.adgroup/README.md:6]: Thanks, that's a good catch! This one was mislabelled.

**Pull request merged**
